app/helpers/notification_subscriptions_helper.rb

```diff
@@ -1,8 +1,8 @@
 module NotificationSubscriptionsHelper
-  def render_notification_subscription_component(subscription_types, journey: nil)
+  def render_notification_subscription_component(subscription_types, extra_attributes = nil, journey = nil)
```

Comment:
Veci ako `extra_attributes` byvaju zvacsa ako posledny parameter s defaultom `{}`, t.j.
```suggestion
  def render_notification_subscription_component(subscription_types, journey = nil, extra_attributes = {})
```

Pozri napr https://apidock.com/rails/ActionView/Helpers/FormTagHelper/form_tag (najdi tam 'Show source')

Reply:
Ja uz tu defaultnu hodnotu riesim vlastne priamo v partiali s ****extra_attributes || Hash.new**, take riesenie sa ti asi nepaci?
`<%= tag.div 'types': form.subscription_types&.join(","), 'aria-live': 'polite', class: 'notification-subscription', **extra_attributes || Hash.new do %>`

Reply:
na urovni helpera je to lepsie, implementujem

Reply:
Hej, takato logika (`**extra_attributes || Hash.new`) do views nepatri, v helpri je to lepsie.